webroot/src/components/Licensee/LicenseeList/LicenseeList.ts

```diff
@@ -182,7 +198,18 @@ class LicenseeList extends Vue {
         }
     }
 
-    async paginationChange() {
+    async paginationChange(firstIdx, lastIdx, prevNext) {
```

Comment:
Can you document these parameters? Not entirely clear to me what they mean/do.

Reply:
Done.